app/views/green_lanes/check_your_answers/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <a href="#" class="govuk-back-link">Back</a>
+    <h1 class="govuk-heading-l">Check your answers</h1>
+
+    <%= render 'green_lanes/shared/summary_card', title: 'About your goods', edit_path: 'EDIT GOOD DETAILS PAGE' %>
+      
+    <h2 class="govuk-heading-m">Category exemptions</h2>
+    
+    <%= render 'category_exemptions', title: 'Category 1 exemptions', category: '1', assessments: @category_one_assessments, edit_path: 'EDIT PATH HERE CAT 1 questions' %>
+    <%= render 'category_exemptions', title: 'Category 2 exemptions', category: '2', assessments: @category_two_assessments, edit_path: 'EDIT PATH HERE CAT 2 questions' %>
+
+    <div class="govuk-button-group">
+      <%= form_with url: green_lanes_result_path(params[:category]), method: :get do %>
```

Comment:
Ah, I see. You're submitting the answers to a show and having a json body in the request object here that are basically URL params.

So if we just make the method and verb a create/POST then I think we're not going to have to do JSON.parse in the result step